src/components/contextual/pages/pool/PoolTransactionsCard/BoostedPoolSwaps/Swaps.vue

```diff
@@ -0,0 +1,79 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useRoute } from 'vue-router';
+import { flatten } from 'lodash';
+
+import usePoolSwapsQuery from '@/composables/queries/usePoolSwapsQuery';
+
+import { FullPool } from '@/services/balancer/subgraph/types';
+
+import Table from './Table.vue';
+import { getPoolAddress } from '@balancer-labs/balancer-js';
+
+/**
+ * TYPES
+ */
+type Props = {
+  pool: FullPool;
+  loading: boolean;
+};
+
+/**
+ * PROPS
+ */
+withDefaults(defineProps<Props>(), {
+  loading: false
+});
+
+/**
+ * COMPOSABLES
+ */
+const route = useRoute();
+
+/**
+ * STATE
+ */
+const id = route.params.id as string;
+const poolAddress = getPoolAddress(id);
+
+/**
+ * QUERIES
+ */
+const poolSwapsQuery = usePoolSwapsQuery(id, {
+  tokenIn_not: poolAddress,
+  tokenOut_not: poolAddress
```

Comment:
this was done to make sure no invest/withdraw types are in... but after our last discussion we will probably remove the swaps page.